src/scripts/create_dummy_reporting_data.py

```diff
@@ -188,8 +188,8 @@ def generate_webhooks_for_target(
 ):
     """Generate Webhooks."""
     if decpection_rate < 0:
-        random.seed()
-        decpection_rate = random.random()
+        random.seed()  # nosec
```

Comment:
You should include a similar comment here about the `# nosec` and can mention that it applies throughout this file.

Reply:
Is `decpectation_rate` a thing, or is that a typo?

Reply:
@dav3r Added a comment at the beginning of that function detailing why all those nosecs exist. This script here is not used in production and is only for generating local data to test with.

Reply:
@zenine07 FYI, our convention is to have the person who initiated a GitHub conversation be the one to resolve it.  That gives the initiator the opportunity to ensure that they are satisfied with the resolution.

The main exception to this is if a code suggestion is made during a review; when you commit that suggestion, GitHub automatically resolves the conversation.  Otherwise, you shouldn't manually resolve a conversation that someone else started.